device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C224O8/buffers_defaults_t1.j2

```diff
@@ -22,10 +22,10 @@
             'spinerouter_leafrouter' : '0m'
        }
 -%}
-{% set ingress_lossless_pool_size =  '144189440' %}
+{% set ingress_lossless_pool_size =  '144129024' %}
```

Comment:
the t1 hwsku doesn't have the service port defined, do we still need to allocate the buffer for them?

Reply:
Hi @r12f , T1 does contain service ports defined. 
please check port_config.ini for it -https://github.com/sonic-net/sonic-buildimage/blob/master/device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C224O8/port_config.ini#L252

Reply:
ouch! and looks like SN5640 also have these ports too. I wonder if we can remove them, since they will not be used at all in this hwsku (as you can tell from the name).

Reply:
actually we probably need to keep them in SN5600, but I wonder if we can remove them in SN5640? these ports are not needed for packet trimming hack as well.